### PR TITLE
Update vtadmin and vtadmin-web docs

### PR DESCRIPTION
**Pull request by notfelineit**
This PR updates the vtadmin and vtadmin-web docs slightly for more clarification.

### CODE REVIEW

File: content/en/docs/15.0/reference/programs/vtadmin-web.md [hunk 1]
```diff
@@ -28,3 +28,10 @@ These environment variables can be passed inline to the `npm run build` command
 [operators_guide]: ../../vtadmin/operators_guide
 [rbac]: ../../vtadmin/role-based-access-control
 [vtadmin_env_ref]: https://github.com/vitessio/vitess/blob/main/web/vtadmin/src/react-app-env.d.ts
+
```
Comment (rsajwani) [inline content/en/docs/15.0/reference/programs/vtadmin-web.md:31]: the view is not coming properly, you can see here 
https://github.com/vitessio/website/blob/b4d61efe7d35d6c213192b1945ee8439649dd0e5/content/en/docs/15.0/reference/programs/vtadmin-web.md

Comment (rsajwani) [reply content/en/docs/15.0/reference/programs/vtadmin-web.md:31]: Screen Shot 2023-01-17 at 8 51 26 AM

File: content/en/docs/16.0/reference/programs/vtadmin-web.md [hunk 2]
```diff
@@ -28,3 +28,10 @@ These environment variables can be passed inline to the `npm run build` command
 [operators_guide]: ../../vtadmin/operators_guide
 [rbac]: ../../vtadmin/role-based-access-control
 [vtadmin_env_ref]: https://github.com/vitessio/vitess/blob/main/web/vtadmin/src/react-app-env.d.ts
+
```
Comment (rsajwani) [inline content/en/docs/16.0/reference/programs/vtadmin-web.md:31]: same as doc 15.0 , the table view is not coming properly ..

**Comment by notfelineit** [comment]
@rsajwani the table renders correctly in the docs website, so I didn't check the .md file in github:
Screen Shot 2023-01-17 at 9 21 15 AM

Updated to render in github as well.

**Comment by rsajwani** [comment]
DCO is failing you might need to rebase with signed-off

**Review by rsajwani — APPROVED** [review]

**Pull request merged**